sample-status-updater-infrastructure/src/main/groovy/life/qbic/samplestatusupdater/update/SampleTrackingServiceConnector.groovy

```diff
@@ -55,8 +59,22 @@ class SampleTrackingServiceConnector implements SampleTrackingService {
                 throw new SampleUpdateException("Could not update sample ${sampleCode}")
             }
         }
-
     }
 
+    @Override
+    def registerSampleStatus(String sampleCode, Status status, Instant timepoint) throws SampleUpdateException {
+        HttpClient client = RxHttpClient.create(service.rootUrl)
+        //TODO this is only a workaround, as the client seems not to prepend the base url.
```

Comment:
what do you mean here?

Reply:
ahhh 😅

Reply:
is it the correct url?

Reply:
@KochTobi is it how we want to continue doing it? If yes I vote for remove the todo for both methods

Reply:
Might be an outdated comment? I am unsure what it tells us so feel free to remove it.